examples/nft_improved_worker_gltf/threejs_worker.js

```diff
@@ -6,6 +6,25 @@ function isMobile() {
     return /Android|mobile|iPad|iPhone/i.test(navigator.userAgent);
 }
 
+const frameLength = 200; // in ms
+const interpolationFactor = 24;
+
+let trackedMatrix = {
+    // for interpolation
+    delta: [
+        0,0,0,0,
+        0,0,0,0,
+        0,0,0,0,
+        0,0,0,0
+    ],
+    interpolated: [
+        0,0,0,0,
+        0,0,0,0,
+        0,0,0,0,
+        0,0,0,0
+    ]
+}
+
 let markers = {
     "pinball": {
         width: 1637,
```

Comment:
A good question: does we need width, height and dpi of the marker?

Reply:
I think we should have width and height. On 8th they provide that. It helps if you want to dynamically size the overlay to the size of the marker.

Reply:
ok thank you for the clarification @ThorstenBux